core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/STDCMPostProcessing.kt

```diff
@@ -233,6 +233,7 @@ class STDCMPostProcessing(private val graph: STDCMGraph) {
         mutableStopData: MutableList<StopTimeData>,
     ): Double {
         var maxTimeDiff = Double.POSITIVE_INFINITY
+        if (lastStopIndexBeforeNode == 0) maxTimeDiff = node.timeData.maxFirstDepartureDelaying
```

Comment:
Hmm. Isn't it always true then? Shouldn't we just init `maxTimeDiff` to `node.timeData.maxFirstDepartureDelaying`?

Reply:
Nah let's keep it that way then.